src/components/DummyData/CarouselSection.js

```diff
@@ -0,0 +1,32 @@
+import stars from "/public/svg/CarouselSection/icons/stars.svg"
+import cl1 from "/public/svg/CarouselSection/clientImages/client1.svg"
+import cl2 from "/public/svg/CarouselSection/clientImages/client2.svg";
+import cl3 from "/public/svg/CarouselSection/clientImages/client3.svg";
```

Comment:
Importing from images in one file and then using in another file can lead to issues (may not work with non public assets).
A direct image path can be passed as a string - "/svg/CarouselSection/clientImages/client3.svg"
Or image url if external link.